README.md

```diff
@@ -16,24 +18,33 @@ Developer environment to run Tonomy ID locally alongside a blockchain node, and
 
 Check `./scripts/install_prerequisits.sh` for way to install this on Ubuntu 20.0.4.
 
-## Use
-
-Get the app running for the first time
-
-```bash
-./app.sh gitinit
-./app.sh install
-./app.sh init
-```
-
-You can then start and stop the services with.
-
-```bash
-./app.sh start
-```
-
-For other features like logging and data reset see
-
-```bash
-./app.sh
-```
+## Integration
+[Integration Repo](https://github.com/Tonomy-Foundation/Tonomy-ID-Integration)
+
+## Run
+
+Follow these steps one by one & read them carefully. Do not rush through it.
+* Clone the repo
+* `git checkout development`
+* go through [this](https://github.com/Tonomy-Foundation/Tonomy-ID-Integration/blob/development/scripts/install_prerequisits.sh) script, line by line, and install the dependencies you don't have.
+* `./app.sh` and read what each command does
+* `./app.sh gitinstall`
+* `./app.sh install`
+* `./app.sh init`
+* Now, you can do one of the following;
+* `./app.sh start`
+* Check out the links shown
+* Scan the QR code with the expo app
+* `./app.sh test` or `./app.sh test all`
+* `./app.sh log eosio`
+* `./app.sh stop` or `./app.sh reset` or `./app.sh reset all`
+
+For visual aid, a recording of a full walkthrough with a junior dev can be found [here](https://www.loom.com/share/f44be75ce80044a08a73c53ea64a3afd)
+
+A recording which explains how it all works, and how to run tests can be found [here](https://www.loom.com/share/8566b834759742309ebc96c74e955767)
+
+* [Tonomy ID](https://github.com/Tonomy-Foundation/Tonomy-ID) - Our React Native cross-platform mobile wallet for public & private EOSIO blockchains.
```

Comment:
Add a heading here ## Repositories ## to make it clear this is about other repositories